### PR TITLE
fix: updated email and name in workflow

### LINKED ISSUE #300: Update github email and name in documentation workflow
### Is your feature request related to a problem? Please describe.
In the documentation workflow, the email and name are not updated with respect to the organization which is against what is mentioned on the official docusaurus website.
(Read the warning message in the screenshot)

Screenshot (74)

### Describe the solution you'd like
Update email and name

### PR DESCRIPTION
**Pull request by vj-codes**
### Description

As per the official documentation on docusaurus website, updated the organization's email and name in workflow.

Fixes #300 

### Type of Change:

- Code
- Quality Assurance
- Documentation

**Code/Quality Assurance Only**
- Bug fix (non-breaking change which fixes an issue)

### How Has This Been Tested?
Locally

### Checklist:

- [ ] My PR follows the style guidelines of this project
- [ ] I have performed a self-review of my own code or materials
- [ ] I have commented my code or provided relevant documentation, particularly in hard-to-understand areas
- [ ] I have made corresponding changes to the documentation
- [ ] Any dependent changes have been merged

**Code/Quality Assurance Only**
- [ ] My changes generate no new warnings
- [ ] New and existing unit tests pass locally with my changes

### CODE REVIEW

**Comment by vj-codes** [comment]
Not sure why the build is failing, I barely changed anything in the code 👀

**Comment by devkapilbansal** [comment]
Hi @vj-codes
I would suggest adding `python-dateutil` in dependencies.

**Comment by vj-codes** [comment]
@devkapilbansal thank you, it worked!
Do review :)

**Review by devkapilbansal — APPROVED** [review]
LGTM 👍 

Thanks @vj-codes for this changes

**Comment by epicadk** [comment]
> Hi @vj-codes I would suggest adding `python-dateutil` in dependencies.

That's odd. Any idea why this happened?

**Review by epicadk — APPROVED** [review]

**Comment by vj-codes** [comment]
@isabelcosta can you please approve this PR, then it can be counted for my hacktoberfest
Thankss in advance:)

File: requirements.txt [hunk 1]
```diff
@@ -15,3 +15,4 @@ python-dotenv==0.14.0
 regex==2020.9.27
 requests==2.24.0
 werkzeug==0.16.1
+python-dateutil==2.8.2
```
Comment (isabelcosta) [inline requirements.txt:18]: is this needed 😅  Can you remove this and then I am happy to approve :)

Comment (vj-codes) [reply requirements.txt:18]: Not really but the build was failing before adding it, shall I revert the change?

Comment (isabelcosta) [reply requirements.txt:18]: alright, then I guess you can leave it :)

**Pull request merged**